src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -0,0 +1,27 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.dto.UserDto;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/v1/user")
+public class UserController {
+
+    @PostMapping
+    public UserDto create(@RequestBody UserDto userDto) {
+        return userDto;
+    }
+
+    @PutMapping
+    public UserDto block(@RequestBody UserDto userDto) {
+        return userDto;
+    }
+
+    @PutMapping
+    public UserDto generateKey(@RequestBody UserDto userDto) {
+        return userDto;
+    }
```

Comment:
Uważam, że metoda generateKey powinna zwracać String z wygenerowanym kodem ;)

Reply:
Skoro metoda jest generateKey, to sie spodziewam, ze cos wygeneruje. Ja bym dal klase UserKeyDto a w nim pole String value.